Statusbar/views/Menu.swift

```diff
@@ -69,28 +102,20 @@ class Menu: NSMenu {
         }
     }
 
-    func updateItems(location: LocationModel) {
-        if(location.lastUpdateDate == nil) {
-            return
-        }
-
-        icon?.isTemplate = true // best for dark mode
-        titleItem.image = NSImage(named: "icon")
-
-        if location.lastUpdateDate == nil { // location was not yet loaded
-            self.addItem(NSMenuItem(title: "network_error".localized, action: nil, keyEquivalent: ""))
-
-        }
+    func showError() {
+        self.addItem(errorItem)
+    }
 
-        //            icon?.template = true // best for dark mode
+    func updateItems(location: Location, lastVisit: LocationVisit?) {
+        locationItem.title = "Location: \(location.name) (\(visitCount) visits today)"
```

Comment:
Should be localised as well right?

Reply:
Converted to localized strings in the latest commits